examples/seeds/simple.yaml

```diff
@@ -2,21 +2,22 @@ apiVersion: eunomia.kohls.io/v1alpha1
 kind: GitOpsConfig
 metadata:
   name: simple-test
+  namespace: gitops-operator
 spec:
   # Add fields here
   templateSource:
-    uri: http://localhost:8000
+    uri: https://github.com/KohlsTechnology/eunomia
     ref: master
-    contextDir: examples/templates
+    contextDir: simple/templates
   parameterSource:
-    contextDir: examples/parameters
+    contextDir: simple/parameters
   triggers:
   - type: Change
   - type: Webhook
     secret: ciao
   - type: Periodic
     cron: "0 * * * *"
   serviceAccountRef: gitops
-  templateProcessorImage: quay.io/kohlstechnology/eunomia-base:v0.0.1
+  templateProcessorImage: quay.io/kohlstechnology/gitops-ocp-templates:v0.0.1
```

Comment:
It looks like this container image does not exist. Probably should be `eunomia-ocp-templates` instead.